src/azure-cli-core/azure/cli/core/profiles/_shared.py

```diff
@@ -139,6 +141,7 @@ def default_api_version(self):
         ResourceType.MGMT_RESOURCE_SUBSCRIPTIONS: '2019-06-01',
         ResourceType.MGMT_NETWORK_DNS: '2018-05-01',
         ResourceType.MGMT_KEYVAULT: '2018-02-14',
+        ResourceType.DATA_SYNAPSE: '2019-11-01-preview',
```

Comment:
Do you need support multi-api?

Reply:
@jsntcy , does multi-api mean that we use different API versions for different endpoints in Python data plane SDKs?  I think this might be true in the future. Currently we only have one API version - `2019-11-01-preview`

Reply:
> Do you need support multi-api?

No. Removed.